app/src/main/java/com/example/dungeontest/NamingScreen.kt

```diff
@@ -163,4 +173,47 @@ fun NamingScreen(drawerState: DrawerState, scope: CoroutineScope, base64EncodedP
             }
         )
     }
+}
+
+@Composable
+fun TextInputField(tokenValue: MutableState<TextFieldValue>, labelText: String) {
```

Comment:
is this being _copied_ from the other file to here? I don't see it being used here. I'm confused

Reply:
if we're going to have one composable called TextInputField in one file and another with the same name in the other file, could you please make each one private (so it's only visible in its own file)?